exercise-8/src/jdbc2/TestConnection2.java

```diff
@@ -0,0 +1,9 @@
+package jdbc2;
+
+import java.sql.*;
```

Comment:
I think you missed this wildcard imports while changing @Kamalesh-Radhakrishnan

Reply:
Sorry I didn't check that file. Now Resolved!